recipes/lap/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "lap" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "f0464186e7f4a29073e3d960e0fe9d0eaed4515ebf936e6d48662c5e76906799" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
```

Comment:
Homepage says it works for 2.7; is there a reason you're skipping it?

Reply:
Sorry, my mistake, fixed